lms/static/js/course_sharing/course_sharing_events.js

```diff
@@ -0,0 +1,28 @@
+/**
+* Module for emitting Course Sharing Events.
+*/
+(function(define) {
+    'use strict';
+
+    define(['jquery', 'logger'], function($, Logger) {
+        return function(courseId) {
+            $(".action-facebook[data-course-id='" + courseId + "']").on('click', function() {
+                // Emit an event telling that the Facebook share link was clicked.
+                Logger.log('edx.course.sharing.share_clicked', {
```

Comment:
also I suggest you move this logic to a single function that is called with the `social_media_site` as a parameter to reduce the chance of the two calls getting out of sync.